packages/components/src/components/AlertDialog/AlertDialogIconWrapper.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Box } from "../../primitives/Box";
+
+export interface AlertDialogIconWrapperProps
+  extends Omit<React.HTMLAttributes<HTMLDivElement>, "color"> {
+  /** The contents of the modal heading. */
+  children: NonNullable<React.ReactNode>;
+}
+
+/** The heading of the modal. */
+const AlertDialogIconWrapper = React.forwardRef<
+  HTMLDivElement,
+  AlertDialogIconWrapperProps
+>(({ children, ...props }, ref) => {
+  return (
+    <Box.div
+      alignItems="center"
+      backgroundColor="colorBackgroundError"
```

Comment:
Will this always be an error color?

Reply:
Good point! I will add variant options here.